deploy.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+
+# abort on errors
+set -e
+
+# fetch version
+if [ "$#" -le 0 ]; then
+    echo "Usage: $0 <version>"
+    exit 1
+fi
+
+VERSION=$1
+DOCS_DIR=docs/docs_page
+
+#clean and copy images
+if [ -d "$DOCS_DIR" ]; then rm -Rf $DOCS_DIR; fi
+mkdir -p $DOCS_DIR/docs/img/
+cp docs/img/* $DOCS_DIR/docs/img/
+
+# build
```

Comment:
What do you think about displaying an error message when jazzy is not installed?

```patch
diff --git a/deploy.sh b/deploy.sh
index 1e6442a..fe01b6d 100755
--- a/deploy.sh
+++ b/deploy.sh
@@ -17,7 +17,11 @@ if [ -d "$DOCS_DIR" ]; then rm -Rf $DOCS_DIR; fi
 mkdir -p $DOCS_DIR/docs/img/
 cp docs/img/* $DOCS_DIR/docs/img/
 
-# build
+# build docs
+if ! [ -x "$(command -v jazzy)" ]; then
+  echo 'Error: jazzy is not installed. You can install it with `gem install jazzy`.' >&2
+  exit 1
+fi
 
 jazzy \
   --objc \
```

Reply:
Also added a warning if jazzy is not in version 0.10.0 (we might have template incompatibilities). See 48243a3